libNeonCore/include/Neon/core/tools/Logger.h

```diff
@@ -53,7 +53,12 @@ LIBNEONCORE_EXPORT extern Logger LoggerObj;
 
 }  // namespace Neon
 
+#if defined( NEON_ACTIVETE_TRACING)
```

Comment:
I think we don't need this. If you want to turn the logging completely off you can just do `Neon::globalSpace::LoggerObj.set_level(spdlog::level::off)`